src/core/opamSystem.ml

```diff
@@ -88,9 +88,9 @@ let rm_command =
     "rm -rf"
 
 let remove_dir dir =
-  log "rmdir %s" dir;
+  log "rmdir %s" (Filename.quote dir);
```

Comment:
FWIW, this one's not necessary - it's just a logging line

Reply:
yeah but i figured it's better to show what's really going to be done